Yosemite/Yosemite/Model/Payments/CardPresentPaymentsConfiguration.swift

```diff
@@ -4,11 +4,13 @@ public struct CardPresentPaymentsConfiguration {
     public let paymentMethods: [WCPayPaymentMethodType]
     public let currencies: [String]
     public let paymentGateways: [String]
+    public let supportedReaders: [CardReaderType]
 
-    init(paymentMethods: [WCPayPaymentMethodType], currencies: [String], paymentGateways: [String]) {
+    init(paymentMethods: [WCPayPaymentMethodType], currencies: [String], paymentGateways: [String], supportedReaders: [CardReaderType]) {
         self.paymentMethods = paymentMethods
         self.currencies = currencies
         self.paymentGateways = paymentGateways
+        self.supportedReaders = supportedReaders
```

Comment:
Should this also be added to `isSupportedReader`?

Reply:
Sorry, I don't see any `isSupportedReader` in the code, I'm not sure what you're referring to 🤔

Reply:
Oh right, I forgot about that. I'll add it though I might replace that method completely on another PR